sdk/core/Azure.Core.Amqp/src/Azure.Core.Amqp.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Azure AMQP types</Description>
+    <AssemblyTitle>Microsoft Azure AMQP</AssemblyTitle>
+    <Version>1.0.0-beta.1</Version>
+    <PackageTags>Microsoft Azure AMQP</PackageTags>
+    <Nullable>enable</Nullable>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <EnableClientSdkAnalyzers>false</EnableClientSdkAnalyzers>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(AzureCoreSharedSources)Argument.cs" Link="SharedSource\Azure.Core\Argument.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\Azure.Core.Experimental\src\Azure.Core.Experimental.csproj" />
```

Comment:
Should this have a dependency on experimental? We need to ensure we don't end up taking a shipping GA dependency on this package.

Reply:
This will be removed once we ship System.BinaryData.